packages/core/test/actor.test.ts

```diff
@@ -352,9 +361,8 @@ describe('spawning callbacks', () => {
         idle: {
           entry: assign({
             callbackRef: ({ spawn }) =>
-              // TODO: this is typed as AnyActorRef instead of CallbackActorRef<{ type: 'START' }>
```

Comment:
this is also not possible to infer from bare `fromCallback` - `{ type: 'START' }` didn't appear anywhere in its signature and contextual types for return positions in nested calls (`assign` within `createMachine`) are sometimes not working (that's why we sometimes have to use `as const` within `raise` etc)